wicketd/src/update_tracker.rs

```diff
@@ -436,6 +438,29 @@ impl UpdateTrackerData {
         Ok(())
     }
 
+    async fn abort_update(
+        &mut self,
+        sp: SpIdentifier,
+        message: String,
+    ) -> Result<(), AbortUpdateError> {
+        let Some(update_data) = self.sp_update_data.get(&sp) else {
+            return Err(AbortUpdateError::UpdateNotStarted);
+        };
+
+        // Is an update not running? If so, then accept the request.
```

Comment:
Having a hard time parsing this line :-) Maybe invert it?

```suggestion
        // We can only abort an update if it is still running.
```

Reply:
Thanks, fixed!